packages/camera/camera/example/lib/main.dart

```diff
@@ -196,7 +189,7 @@ class _CameraExampleHomeState extends State<CameraExampleHome>
 
   Future<void> _handleScaleUpdate(ScaleUpdateDetails details) async {
     // When there are not exactly two fingers on screen don't scale
-    if (_pointers != 2) {
+    if (details.pointerCount != 2) {
```

Comment:
Can't find pointerCount

Reply:
Did some digging around, turns out `ScaleUpdateDetails.pointerCounter` is recently added and is available on the Flutter beta channel. I reverted this change and wait for it to land on stable. Thank you for reporting this issue.